### ISSUE TITLE
Memory leak in mudrod .war artifact

### ISSUE THREAD

**Issue by lewismc** [mudrod/mudrod#56]
Upon analysis of local catalina.out log, I see the following

```
SEVERE: Error listenerStart
Sep 15, 2016 2:01:18 PM org.apache.catalina.core.StandardContext startInternal
SEVERE: Context [/mudrod-service-0.0.1-SNAPSHOT] startup failed due to previous errors
Sep 15, 2016 2:01:18 PM org.apache.catalina.loader.WebappClassLoader clearReferencesThreads
SEVERE: The web application [/mudrod-service-0.0.1-SNAPSHOT] appears to have started a thread named [elasticsearch[Mar-Vell][[timer]]] but has failed to stop it. This is very likely to create a memory leak.
```

There are a number of other issues noted within my local log which I am going to work through however for the time being this one will be dealt with first.

**Comment by Yongyao** [comment]
Yeah, this is the problem I saw before. But it's hard to duplicate. I think it's one of the reasons that caused random shutdown. Thanks, Lewis

**Comment by Yongyao** [comment]
I just realized it is because the ES client created in contextlistener won't be closed if the tomcat is shutdown. 
https://github.com/mudrod/mudrod/blob/temp0907/service/src/main/java/esiptestbed/mudrod/webservlet/MudrodWebListener.java#L56

I should have closed the client in the contextDestroy() method.
https://github.com/mudrod/mudrod/blob/temp0907/service/src/main/java/esiptestbed/mudrod/webservlet/MudrodWebListener.java#L46

@lewismc If you didn't get the chance to do it yet, I can do it. Jut let me know.

**Comment by lewismc** [comment]
yes that is exactly what is happening. Please address this issue and add it to the spark2.0.0 branch thank you.

**Comment by Yongyao** [comment]
Fixed in fb38aa2ddd83f3d94c456af7eb87fd9c49e7348e
